backend/src/services/sync/museum/treasureWriter.test.ts

```
@@ -324,3 +325,113 @@ describe('the works delta a museum run reports', () => {
+ * condition is the photographer's name. What these pin is the half a live run
+ * cannot show: that a run which could not reach Commons resends what the row
+ * already says rather than dropping it, and that it never writes one
+ * photographer's name beside another's photograph.
+ */
+describe('what a run stores about a work photograph', () => {
+  beforeEach(() => {
+    mockedQuery.mockReset();
+    mockedRetire.mockReset();
+  });
+
+  const FILE = 'http://commons.wikimedia.org/wiki/Special:FilePath/Mesha%20stele.jpg';
```

ESLint

[error] Using http protocol is insecure. Use https instead.

(sonarjs/no-clear-text-protocols)